App.js

```diff
@@ -1,11 +1,38 @@
 import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import { StyleSheet, Text, View} from 'react-native';
+import { Font } from 'expo';
+
 
 export default class App extends React.Component {
```

Comment:
Make sure to move this code into its own file. Probably wanna call it `login.js`

Reply:
Do you mean that I should only have code in `login.js`?

Reply:
Yup